backend/codesirius/problems/migrations/0002_alter_language_name_alter_language_version_and_more.py

```
@@ -0,0 +1,31 @@
+            name="name",
+            field=models.CharField(max_length=50),
+        ),
+        migrations.AlterField(
+            model_name="language",
+            name="version",
+            field=models.CharField(max_length=50),
+        ),
+        migrations.AddConstraint(
+            model_name="language",
+            constraint=models.UniqueConstraint(
+                fields=("name", "version"), name="unique_language_name_version"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)